lib/main.coffee

```diff
@@ -9,6 +9,10 @@ module.exports =
         '.source.jade':
           default: false
       description: 'Automatically remove whitespace characters at ends of lines when the buffer is saved. To disable/enable for a certain language, use [syntax-scoped properties](https://github.com/atom/whitespace#readme) in your `config.cson`.'
+    removeTrailingWhitespaceAsYouType:
+      type: 'boolean'
+      default: false
```

Comment:
The current behavior is `true` correct? So this should default to `true`. We don't want to break current default behavior.